aws/modules/deploy/Rakefile

```diff
@@ -130,7 +130,7 @@ do
   end
 
   HELMCHARTS_DIRS.each do |d|
-    rendered = ERB.new(File.read("helms/#{d}-values.erb")).result(binding)
+    rendered = ERB.new(File.read("helms/#{d}-values.#{File.file?("helms/#{d}-values.erb") ? 'erb' : 'yml'}")).result(binding)
```

Comment:
Hmm, I didn't understand how this was set up. Let's get rid of this extra logic and rename the file back to .erb. Sorry for the confusion.

(It would be better if this used the logic below for `rule ".yml" => ".erb"` but let's not mess with that today.)

Reply:
Reverted